### PR TITLE
Screen

### LINKED ISSUE #17: Set up the screen for raspberry pi
Get some life from the screen upon a successful connection.

### PR DESCRIPTION
**Pull request by rymdolle**
Show album cover on screen

### CODE REVIEW

File: src/media_manager.cpp [hunk 1]
```diff
@@ -106,68 +101,138 @@ void media_manager::play()
...
+
+  delete req;
+  req = nullptr;
+}
+
+bool media_manager::is_playing() {
+   return playing;
+}
+
+void media_manager::continue_playing() {
```
Comment (Ecksmas) [inline src/media_manager.cpp]: Pls change continue_playing to currently_playing

**Review by Ecksmas — CHANGES_REQUESTED** [review]
Pls look at my comment

**Review by Ecksmas — APPROVED** [review]
LGTM

**Comment by Lapingold** [comment]
LGTM

**Review by Lapingold — APPROVED** [review]
LGTM

**Pull request merged**